README.md

```diff
@@ -114,3 +114,5 @@ Wheat: https://commons.wikimedia.org/wiki/File:Meuble_bl%C3%A9_tig%C3%A9_feuill%
 Fruit Salad: https://pixabay.com/en/desserts-fruit-salad-bowl-stacked-575437/ (Resized)
 
 Pineapple Pie: http://res.freestockphotos.biz/pictures/17/17487-illustration-of-a-slice-of-pie-pv.png (Resized)
+
+Whisk: https://c1.staticflickr.com/3/2707/4049374496_5809b856f8_b.jpg (Resized)
```

Comment:
This link and the Whisk.png image do not match. As stated earlier please use license compatible images and provide parent page links for such icons in readme.